OpenRA.Mods.Common/Traits/Captures.cs

```diff
@@ -23,6 +23,9 @@ public class CapturesInfo : ITraitInfo
 		[Desc("Types of actors that it can capture, as long as the type also exists in the Capturable Type: trait.")]
 		public readonly HashSet<string> CaptureTypes = new HashSet<string> { "building" };
 
+		[Desc("Triggers TransformOnCapture tagged by these types")]
+		public readonly HashSet<string> TransformTypes = new HashSet<string> { "husk" };
```

Comment:
Can you please define `CaptureTypes` on `TransformOnCapure` instead?  These traits shouldn't need to know about that trait.